common/src/main/resources/META-INF/openapi.json

```diff
@@ -3866,6 +3990,93 @@
                 "example": {
                     "owner": "bwayne"
                 }
+            },
+            "GroupMetaData": {
+                "title": "Root Type for GroupMetaData",
+                "description": "",
+                "required": [
+                    "id",
+                    "description",
+                    "artifactsType",
```

Comment:
This doesn't match any of the properties described below.  How did this happen if you used Studio to edit the API description?  A bug in Studio perhaps?

Reply:
Very likely I manually updated the source in studio...